src/cityreader/cityreader.py

```diff
@@ -17,18 +26,27 @@
 cities = []
 
 def cityreader(cities=[]):
-  # TODO Implement the functionality to read from the 'cities.csv' file
-  # For each city record, create a new City instance and add it to the 
-  # `cities` list
-    
+    # TODO Implement the functionality to read from the 'cities.csv' file
+    with open('cities.csv') as csv_file:
+        csv_reader = csv.reader(csv_file, delimiter= ',')
+        # For each city record, create a new City instance 
+        for row in csv_reader:
+            city = row[0]
+            lat = row[3]
+            lon = row[4]
+            # add it to the `cities` list
+            if row[0] != "city":
```

Comment:
Nice way of doing it! You could also use `next(csv_reader)` right before the for loop as a way to advance one line, which would skip a header regardless of its specific contents